quarkus/config-api/src/main/java/org/keycloak/config/SecurityOptions.java

```diff
@@ -18,6 +18,17 @@ public class SecurityOptions {
             .defaultValue(FipsMode.DISABLED)
             .build();
 
+    public static final Option<String> TRUSTSTORE = new OptionBuilder<>("truststore", String.class)
+            .category(OptionCategory.SECURITY)
+            .description("List of pkcs12 (.p12 or .pfx) or PEM files that will be used to construct the javax.net.ssl.trustStore used at runtime.")
+            .build();
```

Comment:
What about to use the 'truststores' option name as it imply you can specify more of them? Or did I miss some discussion about it?

```suggestion
    public static final Option<String> TRUSTSTORES = new OptionBuilder<>("truststores", String.class)
            .category(OptionCategory.SECURITY)
            .description("List of pkcs12 (.p12 or .pfx) or PEM files that will be used to construct the javax.net.ssl.trustStore used at runtime.")
            .build();
```

Or we could potentially specify the Option as:
```
Option<List<String>> TRUSTSTORES = new OptionBuilder<>("truststores", List.class, String.class)
```
However, it'd require certain other changes such as transformer method in the PropertyMapper. It'd be also good to have some method in the `Configuration` class, which could retrieve a collection of items (could be useful even for Features).

Reply:
@mabartos there was no discussion of singular vs plural. I must have been thinking that singular matched the end result - that is we are creating a singluar truststore.  I'm fine with using the plural truststores though.

> Or we could potentially specify the Option as:

I'd leave that to be a separate refinement to keep these changes focused.